BuildResidentialScheduleFile/measure.rb

```diff
@@ -72,6 +73,12 @@ def arguments(model)
     arg.setDescription('Absolute/relative output path of the HPXML file. This HPXML file will include the output CSV path.')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeBoolArgument('debug', false)
+    arg.setDisplayName('Debug Mode?')
+    arg.setDescription('Applicable when schedules type is stochastic. If true: Write extra state column(s).')
+    arg.setDefaultValue(false)
+    args << arg
```

Comment:
It'd be nice to hook this up to the `run_simulation.rb --debug` flag, so you can do something like:
`openstudio workflow/run_simulation.rb -x workflow/sample_files/base.xml -o test --add-detailed-schedule stochastic --debug`
and then see the sleep column in the CSV.